### ISSUE TITLE
CR 2: Required Rebalancing

### ISSUE THREAD

**Issue by CodeOvrLd** [CodeOvrLd/xFiberCraft#4]
Some items shouldn't be so cheap to make, nor take as short a time as they do. Possible solutions: 
Tiered Items based on (Repair Cost):
- [x]  Tags to assign amounts of plant fibers
- [x]  Item Names, i.e. contains, starts-with, ends-with, to assign amounts of plant fibers
- [x] Alternative method, tbd
Tiered Recipes based on (Crafting Cost): 
- [x]  Tags to assign amounts of plant fibers
- [x]  Recipe Names, i.e. contains, starts-with, ends-with, to assign ingredient amounts of plant fibers
- [x] Alternative method, tbd

**Comment by CodeOvrLd** [comment]
Checking off all the above methods. We found a new way. we figurd out scrapping and now we have made Fiber's for all types of things in the game. we still have some stuff that doesnt fit into those categories so we are still in the process of rebalancing.

**Comment by CodeOvrLd** [comment]
Removed Bundles and Boxes from being crafted with plant fibers, the base items should be mined/harvested and then used to make the bundles/boxes 

Added In Conversion Recipes for ToolFiber and WeaponFiber. Seeing as it should be easy to build a stone axe, wooden club I added in the conversion of 5 plant fibers for 1 ToolFiber. In this way we can allow the user to pick some fiber early on and use that to exchange for better goods. Probably needs a tutorial quest so that we can explain what we are doing in game.
